packages/framer-motion/src/animation/optimized-appear/start.ts

```diff
@@ -65,20 +65,26 @@ export function startOptimizedAppearAnimation(
          */
         window.MotionHandoffAnimation = handoffOptimizedAppearAnimation
 
+        window.MotionHasOptimisedTransformAnimation = (elementId?: string) => {
+            if (!elementId) return false
+
+            const animationId = appearStoreId(elementId, "transform")
+            return Boolean(appearAnimationStore.get(animationId))
+        }
+
         /**
          * We only need to cancel transform animations as
          * they're the ones that will interfere with the
          * layout animation measurements.
          */
-        window.MotionHandoffCancelAll = () => {
-            appearAnimationStore.forEach(({ animation }, animationId) => {
-                if (animationId.endsWith("transform")) {
-                    animation.cancel()
-                    appearAnimationStore.delete(animationId)
-                }
-            })
-
-            window.MotionHandoffCancelAll = undefined
+        window.MotionCancelOptimisedTransform = (elementId: string) => {
+            const animationId = appearStoreId(elementId, "transform")
+            const data = appearAnimationStore.get(animationId)
+
+            if (data) {
+                data.animation.cancel()
+                appearAnimationStore.delete(animationId)
```

Comment:
This is out of scope, but I'm curious why .delete is not handled by `.cancel()` internally? Is there a call site where you want to cancel but not deleting the entry?

Reply:
`animation` is the WAAPI animation that we're keeping in an external store - so it's more like this function is that wrapper function you describe